install/ensure-relay-credentials-test.sh

```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env bash
 source "$(dirname $0)/_test_setup.sh"
+source dc-detect-version.sh
 
-cfg="../relay/config.yml"
-creds="../relay/credentials.json"
+cfg_file="../relay/config.yml"
+creds_file="../relay/credentials.json"
```

Comment:
For the record there is a `creds` defined in `dc-detect-version.sh` that we don't want to clobber. `cfg` changed to match for style.